library/Mockery.php

```diff
@@ -88,12 +88,9 @@ public static function builtInTypes()
             'string',
             'iterable',
             'void',
+            'object',
```

Comment:
What do you think about keeping this list sorted by name?

We can probably also remove the comment `// Up to php 7` that is slightly out of view.